magni_viz/launch/magni_nav.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+  <node pkg="rviz" type="rviz" name="$(anon rviz)"
+        args="-d $(find magni_viz)/rviz/magni_nav.rviz"/>
```

Comment:
Should be `view_nav.rviz` and `view_nav.launch` magni is a little redundant in magni_nav.

Same goes for magni_robot.rviz and magni_robot.launch